plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBKeyValueStorageFactory.java

```diff
@@ -177,19 +181,35 @@ public KeyValueStorage create(
                 segments.stream()
                     .filter(segmentId -> segmentId.includeInDatabaseVersion(databaseVersion))
                     .collect(Collectors.toList());
-
-            segmentedStorage =
-                new RocksDBColumnarKeyValueStorage(
-                    rocksDBConfiguration,
-                    segmentsForVersion,
-                    ignorableSegments,
-                    metricsSystem,
-                    rocksDBMetricsFactory);
+            if (isForestStorageFormat) {
+              LOG.info("FOREST mode detected, using pessimistic DB.");
```

Comment:
I wouldn't call it pessimistic per se 😆 .  maybe locking ?

Reply:
changed to TransactionDB and moved that to the debug level.

Reply:
pessimistic occurs elsewhere in naming (class name)

Reply:
javadoc